doc/nrf/ug_radio_fem.rst

```diff
@@ -61,6 +61,36 @@ Before you add the devicetree node in your application, complete the following s
    The MPSL library provides API to configure FEM.
    See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
 #. Enable support for MPSL implementation in |NCS| by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``.
+#. Enable support for the FEM subsystem in |NCS| by setting the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option to ``y``.
+#. Chose the used FEM implementation by selecting an appropriate Kconfig option.
+
+The following FEM implementations are supported:
+
+* The nRF21540 GPIO implementation, see :ref:`ug_radio_fem_nrf21540_gpio`.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_GPIO` Kconfig option to ``y``.
+* The nRF21540 GPIO SPI implementation, see :ref:`ug_radio_fem_nrf21540_spi_gpio`.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_GPIO_SPI` Kconfig option to ``y``.
+* The nRF21540 2-pin simple GPIO implementation.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_SIMPLE_GPIO` Kconfig option to ``y``.
+
+Setting the FEM output power
+----------------------------
+
+The FEM gain to use by the simple GPIO FEM implementation is provided by the ``tx_gain_db`` property in devicetree and must represent the real gain of the FEM.
+No possibility of control the gain in runtime is provided when using this implementation.
+
+When using nRF21540 implementations the gain to use is set by the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB` (this does not have
+to be set explicitly, if the user is satisfied with the default value of 10).
+This option can only be set to a value of one of the POUTA or POUTB gains, which are provided by the Kconfig options :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA` or :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB`.
+
+Important notice: :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA` and :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB` are by default
+set to 20 and 10 and these are factory precalibrated gain values.
+The user should not change these values, unless POUTA and POUTB were calibrated to different values on special user request.
+
+Runtime gain control of the gain can be enabled, by setting the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL` to y.
+This will cause MPSL to switch the gain between the values POUTA and POUTB, attempting to minimize signal distortions.
+When using the nRF21540 GPIO implementation, the MODE pin must be enabled in devicetree.
+This isn't needed for the GPIO SPI implementation, as the implementation uses the SPI gain instead of the mode pin in order to control the gain.
 
 You can use only the :ref:`nrfxlib:mpsl_fem` API if your application does not require other MPSL features.
 This might be useful when you want to run simple radio protocols that are not intended to be used concurrently with other protocols.
```

Comment:
Line: 105:
Suggest change: `...given a real transmission power at the device output...` What do you think?

Reply:
Line: 106:
Could we describe more examples and add also internal condition to it?
...external conditions such as temperature... -> external or internal conditions (for example temperature, power supply voltage, channel frequency) might affect...

Reply:
I deleted the "the" word, it was there by accident. However I prefer not to write "real transmission power", as it might not be the truth (the RF path may cause attenuation)

Reply:
Regarding the second comment - I'd prefer not to introduce changes to the part about the model, as the focus of this PR is different.